egs2/tedlium2/asr1/asr.sh

```diff
@@ -0,0 +1,1348 @@
+#!/bin/bash
```

Comment:
Please use soft link (e.g. `ln -s`) and refer files in TEMPLATE for both asr.sh and db.sh